### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by sofianehaddad**
predictivity --> predictability

### CODE REVIEW

**Comment by mbaudin47** [comment]
Thank you for the suggestion. 
I could not find any bibliographical reference to this expression. Do you have any pointer?

**Comment by sofianehaddad** [comment]
hmm
[predictivity](https://dictionary.cambridge.org/spellcheck/english-french/?q=predictivity) seems to not exist in english whereas the [second one](https://dictionary.cambridge.org/dictionary/english/predictability) do exist and is applied.

It means also that some methods in the tool should be updated ;-)

File: 2020/Courses/Metamodel/Introduction/steps-C-surrogate.tex [hunk 1]
```diff
@@ -18,7 +18,7 @@
 \item The goal of this course is to introduce the main surrogate methods used in 
 steps C. 
 
-\item Principles of surrogate models, cross-validation, coefficient of predictivity, over-fitting.
+\item Principles of surrogate models, cross-validation, coefficient of predictability, over-fitting.
```
Comment (sofianehaddad) [inline 2020/Courses/Metamodel/Introduction/steps-C-surrogate.tex:22]: we can probably talk about score instead of "coefficient of determination".

File: 2020/Courses/Metamodel/Introduction/steps-C-surrogate.tex [hunk 2]
```diff
@@ -231,7 +231,7 @@ \section{Training and validation}
 Let $g(\bx_v)$ and $\tilde{g}(\bx_v)$ the outputs of the 
 model and metamodel on the validation sample.
 
-The $Q^2$ predictivity coefficient is defined as the $R^2$ 
+The $Q^2$ predictability coefficient is defined as the $R^2$
```
Comment (sofianehaddad) [inline 2020/Courses/Metamodel/Introduction/steps-C-surrogate.tex:234]:
```suggestion
The $Q^2$ coefficient is defined similarly to the $R^2$
```

File: 2020/Scripts/Metamodel/Exercice_Chaos_cantilever_beam_Solution.ipynb [hunk 3]
```diff
@@ -279,7 +279,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Compute the $Q^2$ predictivity coefficient"
+    "Compute the $Q^2$ predictability coefficient"
```
Comment (sofianehaddad) [inline 2020/Scripts/Metamodel/Exercice_Chaos_cantilever_beam_Solution.ipynb:282]:
```suggestion
    "Compute the $Q^2$ score"
```

**Comment by sofianehaddad** [comment]
If we want to be more generic, we can simply use  `score`

**Comment by mbaudin47** [comment]
Here is an attempt of a bibliography.

For R2, the correct wording is "coefficient of determination"[^Wiki].  

In [^Stack], we find that: 
- R2: "goodness of fit", or explained variation. 
- Q2: "goodness of prediction", or predicted variation.

In [^Quan1988], this is called the "Q2 statistics", associated with the "Prediction sum of squares". It refers to PLS, Wold (1982).

In [^Müller2016], the "score" is Scikit-Learn's method. Pages 23-24, it is used in classification, as the proportion of the test set which is correctly classified. 

I am not sure about the conclusion. Isn't "score" more general than it should, including classification instead of the regression calculations performed by OpenTURNS in e.g. he `MetamodelValidation` class?

[^Quan1988]: "The Prediction Sum of Squares as a General Measure for Regression Diagnostics", by Nguyen T. Quan. Journal of Business & Economic Statistics. Vol. 6, No. 4 (Oct., 1988), pp. 501-504. https://www.jstor.org/stable/1391469
[^Müller2016]: Müller, A. C., & Guido, S. (2016). Introduction to machine learning with Python: a guide for data scientists. " O'Reilly Media, Inc.".
[^Wiki]: https://en.wikipedia.org/wiki/Coefficient_of_determination
[^Stack]: https://stats.stackexchange.com/questions/292673/validation-metrics-r2-and-q2-for-partial-least-squares-pls-regression

**Comment by sofianehaddad** [comment]
Yes `score` is more general (both for regression & classification for example). For `regression`, it returns the `Q2` whereas for classification, it returns the mean `accuracy` .

 I think there are several things to keep in mind : 
 - In the `API`, `MetaModelValidation::computePredictivityFactor` should be renamed (`score`  for example). The class is (for the moment) designed only for regression. If we plan to use it for classification, we might face some issues.
 -  In the current `material`, change the `predictivity` by more appropriate words (coefficient of determination, r2 score, goodness of fit/prediction). I will update my PR soon and will discuss it again
